defi/src/protocols/data3.ts

```diff
@@ -47735,7 +47735,7 @@ const data3: Protocol[] = [
     module: "solayer/index.js",
     twitter: "solayer_labs",
     forkedFrom: [],
-    oracles: [],
+    oracles: ["Switchboard"],
```

Comment:
Solayer is using Switchboard for sUSD redemptions:

https://docs.solayer.org/susd/protocol-info/redemption-price#api-overview

Reply:
Regarding solayer, majority of their TVL is liquid staked tokens of SOL , not their stablecoin

Reply:
We have separated out sUSD here onto a separate project at "solayer-susd/index.js".
The new project configuration is now live on DeFiLlama so we can add this to the TVL for just the stablecoin for Switchboard